README.md

```diff
@@ -30,6 +30,33 @@ There are plenty of small, self-contained examples [in the documentation][exampl
 
 The easiest way to get started using ct is to download one of the binaries from the [releases page on GitHub][releases].
 
+One can use the following script to download and verify the signature of Config Transpiler:
+
+```bash
+# Sepcify Config Transpiler version
+CT_VER=v0.6.1
+# Sepcify OS
+OS=apple-darwin # MacOS
+OS=unknown-linux-gnu # Linux
+# Specify download URL
+DOWNLOAD_URL=https://github.com/coreos/container-linux-config-transpiler/releases/download
+
+# Remove previous downloads
+rm -f /tmp/ct-${CT_VER}-x86_64-${OS} /tmp/ct-${CT_VER}-x86_64-${OS}.asc /tmp/coreos-app-signing-pubkey.gpg
```

Comment:
I'd be in favor of downloading to temporary files then using `mv` to overwrite the old ones if everything downloaded and verified ok. That would prevent leaving users with nothing if something fails later. Either that change `-f` to `-i` so users don't accidently delete things they didn't want to.

Reply:
I used the same strategy as here:

![screenshot from 2018-02-05 21-22-23](https://user-images.githubusercontent.com/384489/35826992-c9f810ec-0aba-11e8-81ad-bb4ad9f22da2.png)

`/tmp` is a temporary directory. Stuff inside this directory doesn't survive the reboot. I wouldn't bother about removing files which are critical for the user.
The `-f` flag is used to not throw an error if the file is not available.

What do You think @ajeddeloh ?

Reply:
oh, yeah hadn't thought about it being /tmp SGTM